etc/settings/settings_test.json

```diff
@@ -217,71 +217,24 @@
             }
         },
         "Placement": {
-            "placement_double_spinbox_ex": {
-                "label": "Double Spinbox Example",
-                "widgetType": "DoubleSpinbox",
-                "minVal": 2.4,
-                "maxVal": 11.4,
-                "stepVal": 1.1,
-                "default": 2.5
-            },
-            "placement_int_spinbox_ex": {
-                "label": "Int Spinbox Example",
-                "widgetType": "Spinbox",
-                "minVal": 2.4,
-                "maxVal": 11.4,
-                "stepVal": 1.1,
-                "default": 2.5
-            },
-            "placement_radiobuttons_vert_ex": {
-                "label": "Radio Button Example (vertical/default)",
-                "widgetType": "radiobuttons",
-                "options": [
-                    "b1",
-                    "b2",
-                    "b3"
-                ],
-                "default": "b2"
-            },
-            "placement_radiobuttons_horiz_ex": {
-                "label": "Radio Button Example (horizontal)",
+        "placement_radiobuttons_horiz_ex": {
```

Comment:
"placement_radiobuttons_horiz_ex" should be renamed to something that describes the setting, make sure to update all references to this as well